src/PowerShellEditorServices/Services/TextDocument/SemanticToken.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models.Proposals;
+
+namespace Microsoft.PowerShell.EditorServices.Services.TextDocument
+{
+    internal class SemanticToken
+    {
+        public SemanticToken(string text, SemanticTokenType type, int line, int index, IEnumerable<string> tokenModifiers)
```

Comment:
Index could be confused for an index/offset into the text or the extent text. `column` is a more explicit pairing with `line`

```suggestion
        public SemanticToken(string text, SemanticTokenType type, int line, int column, IEnumerable<string> tokenModifiers)
```